<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>477</width>
    <height>380</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Create Epub</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="lEPortada">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>70</y>
      <width>113</width>
      <height>23</height>
     </rect>
    </property>
    <property name="dragEnabled">
     <bool>false</bool>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>95</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Cover (JPG file):</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pBPortada">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>70</y>
      <width>85</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Browse...</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Title:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>45</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Author:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lETitulo">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>10</y>
      <width>201</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lEAutor">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>40</y>
      <width>201</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>File with dedication:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lEDedicatoria">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>100</y>
      <width>113</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pBDedicatoria">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>100</y>
      <width>85</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Browse...</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>File with initial quote:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pBAnadirCita">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>140</y>
      <width>85</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Add</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pBBorrarCita">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>170</y>
      <width>85</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Remove</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>230</y>
      <width>115</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Files with chapters:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pBAnadirCapitulo">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>230</y>
      <width>85</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Add</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pBBorrarCapitulo">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>260</y>
      <width>85</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Remove</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pBCrearEpub">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>340</y>
      <width>85</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Create Epub</string>
    </property>
   </widget>
   <widget class="QListWidget" name="lWCita">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>140</y>
      <width>201</width>
      <height>81</height>
     </rect>
    </property>
   </widget>
   <widget class="QListWidget" name="lWCapitulo">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>230</y>
      <width>201</width>
      <height>101</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>lETitulo</tabstop>
  <tabstop>lEAutor</tabstop>
  <tabstop>lEPortada</tabstop>
  <tabstop>pBPortada</tabstop>
  <tabstop>lEDedicatoria</tabstop>
  <tabstop>pBDedicatoria</tabstop>
  <tabstop>lWCita</tabstop>
  <tabstop>pBAnadirCita</tabstop>
  <tabstop>pBBorrarCita</tabstop>
  <tabstop>lWCapitulo</tabstop>
  <tabstop>pBAnadirCapitulo</tabstop>
  <tabstop>pBBorrarCapitulo</tabstop>
  <tabstop>pBCrearEpub</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
